### ISSUE TITLE
[Push Publishing] : Pushing an archived piece of content is being deleted in receiver

### ISSUE THREAD

**Issue by jcastro-dotcms** [dotCMS/core#17994]
Reported via Support ticket: [100462](https://dotcms.zendesk.com/agent/tickets/100462)
Reproducible in dotCMS 4.3.3, and latest master (5.2.6).

The purpose of this ticket is two-fold:

#### Issue # 1
The `Push Now` actionlet is always Push Publishing the specified Contentlet for publishing, but not for archiving. We need to change this to take the current status of the Contentlet and update it in the receiving environment appropriately.

### Steps to reproduce the behavior:
1. Set up a Push Publishing environment.
2. Setup a Workflow Action that calls the `Push Now` actionlet.
3. Create an HTML Page, and push it to the receiver. Make sure it works.
4. Mark the HTML Page as archived, and call the Workflow Action that calls the `Push Now` actionlet.
5. The Contentlet is never marked as archived in the receiving environment.

### Expected behavior:
The `Push Now` actionlet must work with archived Contentlets as well.

#### Issue # 2
The default Push Publishing mechanism allows you to push archived content. However, it ONLY allows you to remove the selected archived content in the receiving environment. The user cannot select any other choice.

This means that, if you have "Content 1" as archived in your sender, and push publish it to the receiver, then "Content 1" will be completely deleted in the destination instead of just being flagged as archived.

### Steps to reproduce the behavior:
1. Set up a Push Publishing environment.
2. Create an HTML Page, and push it to the receiver. Make sure it works.
3. Mark the HTML Page as archived, and push it. Notice how only the "Remove" option is automatically selected, which is wrong.
4. The HTML Page in the receiver has been completely deleted.

### Expected behavior
Push Publishing an archived page MUST NOT result in the page being deleted from the receiver. Also, the Push Publishing dialog must allow the user to select either "Push" or "Remove", not just "Remove".

### Additional context
This problem has been present for a very long time.

## Acceptance Criteria

1. Multilanguage
2. Unit test

**Comment by jcastro-dotcms** [comment]
Hotfix for dotCMS 4.3.3: https://github.com/dotCMS/support/commit/88e0a1899c80f14f65e5b4a204bc3ce7c0e52808

**Comment by jcastro-dotcms** [comment]
Core PR: https://github.com/dotCMS/core/pull/18048
EE PR: https://github.com/dotCMS/enterprise-2.x/pull/701

**Comment by fabrizzio-dotCMS** [comment]
I was able to verify the two scenarios using a FileAsset since at the moment this Bug was verified I discovered an issue that prevents push-publishing new pages. However, I was able to determine the new issue is completely separate from this fix. Therefore I am stamping the Passed internal QA Label.

**Comment by bryanboza** [comment]
This is blocked by #18126

**Comment by bryanboza** [comment]
We still having problems trying to push archived pages, when you try to push a complete folder with an archived page, we have an error and the folder is not pushed 
ST: https://gist.github.com/bryanboza/1d60d5471f1ca6ff8832079370bb9c65

**Comment by bryanboza** [comment]
Fixed, tested on release-5.2.8 // Postgres // FF
